### ISSUE TITLE
files indicated in OpenFiles list in skeleton.dcf files of project template does not open anymore

### ISSUE THREAD

**Issue by MaelTheuliere** [rstudio/rstudio#7940]
### System details

    RStudio Edition : Desktop
    RStudio Version : 1.3.1093
    OS Version      : #52-Ubuntu SMP Thu Sep 10 10:58:49 UTC 2020
    R Version       :  3.6.3

### Steps to reproduce the problem
Install bookdown
Create a Project => New Project => Bookdown Project
### Describe the problem in detail
The files indicated in `OpenFiles:` in the [skeleton.dcf](https://github.com/rstudio/bookdown/blob/master/inst/rstudio/templates/project/skeleton.dcf) file doesn't open anymore at the project creation
### Describe the behavior you expected
The files indicated in `OpenFiles:` in the skeleton.dcf](https://github.com/rstudio/bookdown/blob/master/inst/rstudio/templates/project/skeleton.dcf) file should be open at the project creation : 

- README.md
- _bookdown.yml
-  _output.yml
- index.Rmd

Same behavior found with [golem package project template](https://github.com/ThinkR-open/golem/blob/dev/inst/rstudio/templates/project/create_golem.dcf)

- [x] I have read the guide for [submitting good bug reports](https://github.com/rstudio/rstudio/wiki/Writing-Good-Bug-Reports).
- [x] I have installed the latest version of RStudio, and confirmed that the issue still persists.
- [ ] If I am reporting a RStudio crash, I have included a [diagnostics report](https://support.rstudio.com/hc/en-us/articles/200321257-Running-a-Diagnostics-Report).
- [x] I have done my best to include a minimal, self-contained set of instructions for consistently reproducing the issue.

**Comment by ronblum** [comment]
@MaelTheuliere Thank you for raising this issue!

I can reproduce this in RStudio Desktop 1.3.1056, 1.3.1093 and 1.4.869 on MacOS 11 Big Sure beta 8, as well as RStudio Server 1.4.869 on Red Hat 8.2. This is a regression.

Note: This works properly in RStudio Desktop 1.2.5042 on MacOS.

**Comment by mikebessuille** [comment]
Unfortunately, we are very close to 1.4 release; this is a regression from 1.2, and there is a clear workaround. Moving this to 1.5.

**Comment by kevinushey** [comment]
Looks like this was broken by https://github.com/rstudio/rstudio/commit/05f2af6940f12df057f0094690c35ebdecff810a. The scratch path is now expected as a parameter, but R-based calls were not updated to use the explicit scratch path.

```
$ rg "rs_addFirstRunDoc"
src/cpp/session/modules/SessionProjectTemplate.R
80:   .Call("rs_addFirstRunDoc", projectFilePath, files)

src/cpp/session/modules/SessionPackages.R
1187:      .Call("rs_addFirstRunDoc", RprojPath, "R/hello.R", PACKAGE = "(embedding)")
1198:         .Call("rs_addFirstRunDoc", RprojPath, "src/rcpp_hello.cpp", PACKAGE = "(embedding)")
```

**Comment by kfeinauer** [comment]
Verified fix in version 1.4.944
